Cargo.toml

```diff
@@ -41,6 +41,7 @@ feat_common_core = [
   "renice",
   "rev",
   "setsid",
+  "findmnt",
```

Comment:
This should be in alphabetical order :)

Reply:
Maybe the project should consider adding [cargo sort](https://github.com/DevinR528/cargo-sort) to the CI.